src/pages/Offer/sections/HedvigInfo.tsx

```diff
@@ -69,15 +68,14 @@ const Row = styled('div')({
   },
 })
 
-const Title = styled('h3')({
+const Title = styled('p')({
```

Comment:
this looks weird to me. Why are we using a title as a `p`?

Reply:
It should be a p size wise, maybe title is the wrong naming though

Reply:
i think if it is a title semantically we should just restyle it to be a P

Reply:
Hmm, not sure what you mean🤔 there is very little styling done here, so lets leave it as a `p`?

Reply:
or did you want a `div`?